src/@chakra-ui/gatsby-plugin/components/Modal.ts

```diff
@@ -11,57 +11,67 @@ const {
   defaultProps,
 } = modalDefaultTheme
 
-const variantCode = definePartsStyle((props) => ({
-  overlay: {
-    bg: "rgba(0, 0, 0, 0.7)",
-  },
-  dialog: {
-    maxW: "100vw",
-    marginTop: "auto",
-    marginBottom: 0,
-    maxHeight: "50%",
-    borderRadius: 0,
-  },
-  header: {
-    bg: props.colorMode === "dark" ? "rgb(25, 25, 25)" : "rgb(247, 247, 247)",
-    borderColor:
-      props.colorMode == "dark" ? "rgb(242, 242, 242)" : "rgb(51, 51, 51)",
-    borderTop: "1px solid",
-    borderBottom: "1px solid",
-    textTransform: "uppercase",
-    fontWeight: "normal",
-    fontSize: "md",
-    fontFamily: "monospace",
-  },
-  closeButton: {
-    padding: 0,
-    width: "24px",
-    height: "24px",
-    borderRadius: 0,
-    color: "rgb(178, 178, 178)",
-    fontSize: "sm",
-    margin: 0,
-    top: 4,
-    right: 4,
-    bottom: 4,
-  },
-  body: {
-    padding: 0,
-  },
-}))
+// Remove all the comment line once the review is completed & PR will able to merge
+
+/* const variantCode = definePartsStyle((props) => ({
+ 
+})) */
 
 export const Modal = defineMultiStyleConfig({
   baseStyle: definePartsStyle((props) =>
     defineMergeStyles(defaultBaseStyle?.(props), {
+      overlay: {
+        bg: "rgba(0, 0, 0, 0.7)",
+      },
       dialog: {
-        my: 0,
+        size: "2xl",
+        /* maxW: "464px", */
+        my: "64px",
+        mx: "8px",
+        gap: 4,
+        padding: "32px",
+        maxHeight: "70%",
+        borderRadius: "4px",
+      },
+      header: {
+        /* bg:
+          props.colorMode === "dark" ? "rgb(25, 25, 25)" : "rgb(247, 247, 247)", */
+        borderColor:
+          props.colorMode == "dark" ? "rgb(242, 242, 242)" : "rgb(51, 51, 51)",
+        /* borderTop: "1px solid",
+        borderBottom: "1px solid", */
+        padding: 0,
+        fontWeight: "bold",
+        fontSize: "2xl",
+        fontFamily: "Inter",
+      },
+      closeButton: {
+        padding: 0,
+        width: "24px",
+        height: "24px",
+        borderRadius: 0,
+        color: "rgb(178, 178, 178)",
+        fontSize: "sm",
+        marginTop: 5,
+      },
+      body: {
+        padding: 0,
+        fontWeight: "normal",
+        fontSize: "md",
+        fontFamily: "Inter",
+      },
+      footer: {
+        gap: "8px",
+        px: 0,
+        pt: 8,
+        pb: 0,
       },
     })
   ),
   sizes: defaultSizes,
-  variants: {
+  /* variants: {
     code: variantCode,
-  },
+  }, */
   defaultProps: {
     ...defaultProps,
     variant: undefined,
```

Comment:
I think all of this should be removed. Per the DS, it is suggested that we will only be using one size and style of modal.

However, a couple of the `size` are being used in the `QuizzesModal` component. So do not remove for now!

---
For @pettinarip  below:

I stated in another comment about having to hard-code the `maxWidth` value of the dialog part because the value used in the DS does not have a corresponding size token that is available.

The `QuizzesModal` component is using a size value that is larger than in the DS.

And if we keep the `sizes` object from the default theme as well as the default `size` being used, it will override the hard-coded `maxW` value in the dialog part.

Does this make sense?